webservice/dd_split_types/drainingn.py

```diff
@@ -21,8 +21,13 @@ def __init__(self, ctx, cs, test=False):
         self.cs = cs
         self.db = ctx.db
         self.cs.data_dispatcher_dataset_only = False
-        self.dmr_service = ctx.dmr_service  if ctx.dmr_service else shrek.DMRService(cherrypy.session.get("Shrek", {}))
+        self.dmr_service = ctx.dmr_service  if ctx.dmr_service else shrek.DMRService()
         self.dmr_service.initialize_session(ctx)
+        self.dmr_service.set_data_dispatcher_client()
+        self.dmr_service.set_metacat_client()
```

Comment:
Is class drainingn only relevant to Shrek, or it also used for the SAM case.   If so, will the unconditional set_data_dispatcher_cline() and set_metacat_client() cause issues here?